### PR TITLE
Dcwc 1683 a11y

### PR DESCRIPTION
**Pull request by shewood**
### Description
Fixed A11y errors

List of proposed changes:
Moved the DSHeader and DSFooter into the layout that contains Meta (title etc) and main (landmarks)
Wrapped each page content with the layout.

### CODE REVIEW

**Comment by CurtisUnderwood** [comment]
Also, are we still using NEXT_PUBLIC_ADOBE_ANALYTICS_URL on the Layout? It's been removed from the env example and seems like none of that code is being called

**Comment by shewood** [comment]
Tom, correct me if I am wrong but yes, we still use AA but not AEM.

File: components/organisms/Layout.js [hunk 1]
```diff
@@ -171,5 +190,5 @@ Layout.propTypes = {
 
 Layout.defaultProps = {
   displayHeader: true,
-  displayFooter: true,
+  displayDSFooter: true,
```
[the remark this answers is not in the document]
Comment (shewood) [reply components/organisms/Layout.js:193]: There are actually two different ones.
We need both

File: pages/500.js [hunk 2]
```diff
@@ -1,24 +1,36 @@
...
+        displayHeader={true}
+        displayDSFooter={false}
+        metadata={props.metadata}
+        langToggleLink={props.langToggleLink}
+      >
+        <ErrorPage
+          errType="500"
+          lang={props.locale}
+          accountPageLink={errorPageLink}
+          isAuth={true}
```
[the remark this answers is not in the document]
Comment (shewood) [reply pages/500.js:19]: I didn't make any changes to these other than wrapping the page in meta and main

File: pages/dashboard.js [hunk 3]
```diff
@@ -57,147 +57,160 @@ import {
...
+      <Layout
+        locale={props.locale}
+        displayPhase={false}
+        displayHeader={true}
+        displayDSFooter={true}
+        metadata={props.metadata}
+        langToggleLink={props.langToggleLink}
+      >
+        <LayoutContainer>
+          <div className="col-span-12 mb-8">
```
[the remark this answers is not in the document]
Comment (s-laugh) [reply pages/dashboard.js:83]: I think fixing this in another PR would be a good idea!

**Review by s-laugh — APPROVED** [review]
Great work!

**Review by DaceyTom2 — COMMENTED** [review]
Looks great, one thing I'd recommend is adding layout to _app so it applies to all pages in a future PR

**Pull request merged**